src/main/java/io/cdap/plugin/http/source/common/BaseHttpSourceConfig.java

```diff
@@ -25,7 +25,7 @@
 import io.cdap.plugin.common.ReferencePluginConfig;
 import io.cdap.plugin.http.source.common.error.ErrorHandling;
 import io.cdap.plugin.http.source.common.error.HttpErrorHandlerEntity;
-import io.cdap.plugin.http.source.common.error.HttpErrorHandlingStrategy;
+import io.cdap.plugin.http.source.common.error.RetriableErrorHandling;
```

Comment:
nit: elsewhere in the CDAP codebase we've spelled this as Retryable. I don't think there's a canonical way to spell it, but let's use Retryable instead of Retriable for consistency.

Reply:
Done. Once this is merged can you please also merge #13 as well.